### ISSUE TITLE
String properties > 100 characters fail on ActiveNode 'inspect' 

### ISSUE THREAD

**Issue by louspringer** [neo4jrb/neo4j#1111]
Test case, simple ActiveNode class with String.

File t.rb contains
```
require 'neo4j'

class T
  include Neo4j::ActiveNode
  property :s, type: String
end

Neo4j::Session.open(:server_db, 'http://localhost:7474')
puts 'environment loaded'
```

Load irb

```
$ irb
irb(main):001:0> load 't.rb'
environment loaded
=> true
irb(main):002:0> t = T.new
=> #<T uuid: nil, s: nil>
irb(main):003:0> t.s = '0'*100
=> "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
irb(main):004:0> t
=> #<T uuid: nil, s: "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000">
irb(main):005:0> t.s = '0'*101
=> "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
irb(main):006:0> t
(Object doesn't support #inspect)
=> 
```

The issue appears to be line 54 in active_node/property.rb

```
...
attribute_pairs = attributes.except(id_property_name).sort.map do |key, value|
    [key, (value.is_a?(String) && value.size > 100) ? value.dup.first(100) : value]
end
...
```
'first' is not a method on String. 

Maybe the line should read like this? 

```
[key, (value.is_a?(String) && value.size > 100) ? value.dup[0..100] : value]
```

**Comment by cheerfulstoic** [comment]
Thanks for reporting this!  Unfortunately, I'm not able to reproduce...  What version of `neo4j` are you using?

**Comment by cheerfulstoic** [comment]
Thanks for reporting this.  Was discussed on Gitter and we found that the use of `first` is from Rails.  Changing to `[0..100]` fixes for non-Rails.  Released as 6.0.6 and 6.1.2
